### PR TITLE
Read password from file for Beeline

### PR DESCRIPTION
**Pull request by dekarlab**
Activate option -w password file for beeline for kylin.

Example:
https://cwiki.apache.org/confluence/display/Hive/HiveServer2+Clients
% beeline -u jdbc:hive2://localhost:10000/default -n scott -w password_file

### CODE REVIEW

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

File: source-hive/src/main/java/org/apache/kylin/source/hive/BeelineHiveClient.java [hunk 1]
```diff
@@ -27,6 +27,7 @@
 import java.sql.Statement;
 import java.util.List;
 import java.util.Properties;
+import java.io.*;
```
Comment (shaofengshi) [inline source-hive/src/main/java/org/apache/kylin/source/hive/BeelineHiveClient.java]: I think "import *" is not allowed in Kylin's static code analysis. Could you please change to use the detailed class import?

Comment (nichunen) [reply source-hive/src/main/java/org/apache/kylin/source/hive/BeelineHiveClient.java]: You can set up your kylin development environment by following
http://kylin.apache.org/development/dev_env.html

**Comment by nichunen** [comment]
Jenkins, test this please.

**Comment by nichunen** [comment]
@dekarlab Hi, your pr build failed due to
[ERROR] Forbidden class/interface use: java.io.FileReader [Uses default charset]
[ERROR]   in org.apache.kylin.source.hive.BeelineHiveClient (BeelineHiveClient.java:69)
reliance on default encoding of reader is not recommended in Kylin

**Pull request merged**